runtime/server/api.go

```diff
@@ -64,15 +64,22 @@ func (s *Server) apiHandler(w http.ResponseWriter, req *http.Request) error {
 		return httputil.Error(http.StatusInternalServerError, err)
 	}
 
-	// TODO: Should it resolve security and check access here?
+	resolverProps := api.Spec.ResolverProperties.AsMap()
+	skipResolverSecurity := resolverProps["skip_nested_security"]
+	claims := auth.GetClaims(ctx).SecurityClaims()
+	// for each claim skip checks
+	if skipResolverSecurity != nil && skipResolverSecurity.(bool) {
+		// SkipChecks
+		claims.SkipChecks = true
+	}
```

Comment:
Referencing my comment above – then you'd be able to just have something like this:
```suggestion
	// Rewrite the claims before passing them to the resolver
	claims := auth.GetClaims(ctx).SecurityClaims()
	if api.Spec.SkipNestedSecurity {
		claims.SkipChecks = true
	}
```